previews/primer/beta/button_group_preview/with_menu_button.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render(Primer::Beta::ButtonGroup.new) do |component| %>
+  <% component.with_button { "Main menu" } %>
+  <% component.with_menu_button(button_arguments: { "aria-label": "secondary menu" }, menu_arguments: { anchor_align: :end }) do |menu, button| %>
+    <% menu.with_item(label: "Item 1", item_id: :item1) %>
+    <% menu.with_item(label: "Item 2", item_id: :item2) %>
+    <% button.with_trailing_visual_icon(icon: "triangle-down") %>
```

Comment:
Should we force this to always be an icon button? and require the developer to pass in which icon?